context/app/markdown/docs/metadata.md

```diff
@@ -2,38 +2,73 @@
 
 HuBMAP metadata documentation and submission specifications
 
-## Donor
```

Comment:
This section should remain until it is clear that this won't be delivered.

Reply:
Reverted. Thanks!